src/Controllers/usersController.ts

```diff
@@ -0,0 +1,109 @@
+import { Request, Response } from 'express';
+import db from './../db';
+import { v4 as uuidv4 } from 'uuid';
+import { validationResult } from 'express-validator';
+
+export const createUser = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+  try {
+    const id = uuidv4();
+    console.log("id", id);
+    const login = req.body.login;
+    const password = req.body.password;
+    const age = req.body.age;
+    const isDeleted = false;
+    db.run(`INSERT INTO users(id, login, password, age, isDeleted) VALUES(?, ?, ?, ?, ?)`, [id, login, password, age, isDeleted], function(err) {
+      if (err) {
+        throw err;
+      }
+      res.status(200).json({insertedId: this.lastID});
+    });
+  } catch (e) {
+    res.status(500).json(e);
+  }
+}
+
+export const deleteUser = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({ errors: errors.array() });
+  }
+  try {
+    const id = req.params.id;
+    const isDeleted = true;
+    db.run(`UPDATE users SET isDeleted = ? WHERE id = ?`, [isDeleted, id], function(err) {
+      if (err) {
+        throw err;
+      }
+      res.status(200).json({updated: this.lastID});
```

Comment:
You can create enum with response statuses and use it across all of your application.
`enum HttpStatusCodes{
 ok = 200,
 created = 201,
 ...etc`.